### DISCUSSION TITLE
Support for Orebfuscator

### DISCUSSION THREAD

**Discussion by NgLoader** [BG-Software-LLC/WildTools#92] [Pending Feature Requests]
Can you please provide a hook for [Orebfuscator](https://github.com/Imprex-Development/Orebfuscator)

**Comment by OmerBenGera** [comment]
Paper's anti-xray is a thing and it's supported.

**Reply by NgLoader** [reply]
Paper anti-xray has not everything that Orebfuscator has, like Proximity Hider

**Reply by OmerBenGera** [reply]
> Paper anti-xray has not everything that Orebfuscator has, like Proximity Hider

Ah, I understand.
Not sure if its possible to support it, will look into it.
